src/Workspaces/Core/Portable/Shared/Extensions/ISymbolExtensions.cs

```diff
@@ -101,13 +104,14 @@ public static ImmutableArray<ISymbol> ExplicitInterfaceImplementations(this ISym
             }
         }
 
-        public static ImmutableArray<T> ExplicitOrImplicitInterfaceImplementations<T>(this T symbol) where T : ISymbol
+        public static ImmutableArray<TSymbol> ExplicitOrImplicitInterfaceImplementations<TSymbol>(this TSymbol symbol)
+            where TSymbol : ISymbol
         {
             var containingType = symbol.ContainingType;
-            var allMembersInAllInterfaces = containingType.AllInterfaces.SelectMany(i => i.GetMembers(symbol.Name));
+            var allMembersInAllInterfaces = containingType.AllInterfaces.SelectMany(i => i.GetMembers().OfType<TSymbol>());
```

Comment:
why was this done?  this seems strange that we're not using the name.

Reply:
This was broken for VB and explicit interface implementations, where GetMembers(name) will fail to find implementing members. I have added unit tests for these cases.

---
In reply to: [230607030](https://github.com/dotnet/roslyn/pull/30777#discussion_r230607030) [](ancestors = 230607030)

Reply:
Good catch!  And thanks for the tests! :)